snapshotserver/snapshot.go

```diff
@@ -479,6 +479,7 @@ func DownloadSnapshot(
 	err := GetTenantSnapshotData(scopes, mediaType, db, w)
 	if err != nil {
 		log.Errorf("GetTenantSnapshotData error: %v", err)
+		sendAPIError(http.StatusInternalServerError, err.Error(), w, r)
```

Comment:
Yeah, I think "cant' reach the database" is an OK reason to return 500 (which is what that is.)